frontend/src/components/SupportButton/styles.scss

```
@@ -0,0 +1,8 @@
+@use "../styles/variables/color.scss";
+@use "../styles/variables/font.scss";
+
+.text-button-style {
+  font-family: font.$font-stack;
+  font-size: 24px;
+  color: color.$text-color;
+}
```

Could you refactor this file?

`@use` => `@import`

```suggestion
@use "../styles/variables/color.scss";
@use "../styles/variables/font.scss";

.text-button-style {
  font-family: $font-stack;
  font-size: 24px;
  color: $text-color;
}
```